performance/backend/backend-performance-tools.md

```diff
@@ -0,0 +1,70 @@
+# Back-end performance tools and technics
+
+- **Owner:** [Xavier Lefèvre](https://www.github.com/xavierlefevre)
+- **Last update date:** 3rd of April 2018
+
+## K6: Load-test your back-end with JavaScript written tests
+[K6 official website](https://k6.io/)
+Describe your routes test in JS and run them with simultaneous calls.
+
+## Postman: Document and test the global performance of your routes with an interface
+[Our BNP AM Postman configuration](https://github.com/theodo/bnp-postman)
+Test your routes from always the same software and network, to have a point of comparison.
+
+### iPhone: Simulate a network like LTE, 3G or Edge
```

Comment:
You can also install network link conditioner on your mac too (for simulators, it's easier to manipulate than going to the iPhone settings)